packages/emitter-framework/src/csharp/components/class/declaration.test.tsx

```diff
@@ -73,6 +73,7 @@ it("renders a class declaration with properties using component override", async
       Prop1: string;
       Prop2: int32;
       Prop3?: Foo;
+      Prop4?: "foo" | "bar";
```

Comment:
Before this change this would throw not supported and there is no way to override it